perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -472,6 +474,22 @@ sub inspect : Private {
                             $c->cobrand->max_detailed_info_length
                         );
                 }
+            } elsif ( $c->get_param('detailed_information') eq '' ) {
+                $problem->unset_extra_metadata('detailed_information');
+            }
+
+            my $handler
+                = $c->cobrand->call_hook(
+                    get_body_handler_for_problem => $problem
+                ) || $c->cobrand;
+            my $record_extra
+                = $handler->call_hook('record_update_extra_fields');
+            if (
+                $record_extra->{detailed_information}
+                && ( $problem->get_extra_metadata('detailed_information') // '' )
+                ne $old_detailed_information
+            ) {
+                $update_params{extra}{detailed_information} = 1;
```

Comment:
Thanks ^.^ I didn't think it mattered - and trying to consolidate them into one update seemed like it would be a headache, given that the assigned user logic kind of exists in its own bubble.

Reply:
Am still wondering, though, what I should do about the updates displaying on the /report/<id> page :-/

Reply:
You can see in report/updates.html there's a "IF update.get_extra_metadata triage_report" type thing to skip those updates. Perhaps we take this opportunity to add a `staff_only` flag to updates (for there as well) and have it use that to skip these updates.